serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md

```diff
@@ -0,0 +1,238 @@
+# TensorRT-LLM manual compilation of models tutorial
+
+## Overview
+
+With LMI TensorRT-LLM container, you can run manual compilation of some models and quantization that LMI not supported for JIT and then load the model and run inference.
+
+The goal of this document is for the user to be able to:
+
+1. Convert any TensorRT-LLM supported model a model format that LMI can load and run inference.
+2. Upload the model to S3 so you could use it for runtime.
+
+## Step by step tutorial
+
+In this tutorial, we will be converting the baichuan model to TensorRT-LLM model format on p4d.24xlarge.
+
+### Step 1: Choose your instance
+
+To do the model compilation you need to use the instance that has **the same GPU architecture** that will be used for your inference deployment (e.g model compiled on a g5 instance can be deployed only on g5 instances).
+
+### Step 2: Pull the docker image
+
+Refer [here](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers) for the latest TensorRT-LLM DLC and pull the image.
+
+For example:
+
+```
+docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/djl-inference:0.25.0-tensorrtllm0.5.0-cu122
+```
+
+You can also pull the container from DockerHub:
+
+```
+docker pull deepjavalibrary/djl-serving:0.25.0-tensorrt-llm
+```
+
+### Step 3: Login the container and prepare the environment
+
+You need to manually login into the container to proceed for conversion
+
+```
+docker run -it --runtime=nvidia --gpus all \
+--shm-size 12g \
+deepjavalibrary/djl-serving:0.25.0-tensorrt-llm \
+/bin/bash
+```
+
+Then clone the TensorRT-LLM repository inside the container for manual conversion.
```

Comment:
I think we should provide instruction on how to use our partition script to save the compiled model.

Reply:
We will eventually use partiton script to support it. But for now, we have to manually teach customers